index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="css/reset.css" type="text/css" />
+	<link rel="stylesheet" href="css/style.css" type="text/css" />
+	<title>Klondike</title>
+</head>
+
+<body>
+	<input id="button" type="button" class="button" value="New Game">
+	<div>
+		<div id="top">
+			<div id="div1" class="borders">
```

Comment:
from my point of view it could be easier to use more exact ids, not just 'div-1', but for example 'top-1' here.